pages/common/lpr.md

```diff
@@ -0,0 +1,31 @@
+# lpr
+
+> CUPS tool for printing files.
+
+- Print a file to the default printer:
+
+`lpr {{path/to/file}}`
+
+- Print a certain number of copies:
+
+`lpr -# {{copies}} {{path/to/file}}`
+
+- Print to a named printer:
```

Comment:
How does one list the printers ?

Reply:
It relies on a different tool, so I am not sure if it is appropriate to include here? `lpstat -a` will do it, and is from the same set of tools, but not the same command.

Reply:
We can always include a "see also" which points to lpstat. Something like [this](https://github.com/tldr-pages/tldr/blob/75d109448a21c8bd82e6cc1f2912d8b0d2a8d631/pages/linux/sinfo.md).

Reply:
Added, along with lpadmin (as it seems to be of equivalent pertinence). Also, a link to cups.org for completeness, in case just writing CUPS isn't enough